extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -146,62 +146,47 @@ describe.configure().retryOnSaucelabs().run('amp-bind', function() {
     });
   });
 
+  /**
+   * Combined test specs for amp-img
+   */
   describe('amp-img integration', () => {
-    it('should change src when the src attribute binding changes', () => {
-      const button = fixture.doc.getElementById('changeImgSrcButton');
+    it('should do amp-img integration', () => {
+      const changeImgSrcButton = fixture.doc
+          .getElementById('changeImgSrcButton');
+      const invalidSrcButton = fixture.doc.getElementById('invalidSrcButton');
+      const ftpSrcButton = fixture.doc.getElementById('ftpSrcButton');
+      const changeImgAltButton = fixture.doc
+          .getElementById('changeImgAltButton');
+      const changeImgDimensButton = fixture.doc
+          .getElementById('changeImgDimensButton');
       const img = fixture.doc.getElementById('image');
       expect(img.getAttribute('src')).to.equal('http://www.google.com/image1');
-      button.click();
+      expect(img.getAttribute('height')).to.equal('200');
+      expect(img.getAttribute('width')).to.equal('200');
+      changeImgSrcButton.click();
```

Comment:
I think the goal we're going for here is to try to trigger a change and then verify that it does what we expect. For instance, we expect clicking `changeImgSrcButton` to change the image source, so after it's clicked, we want to verify that it has changed. However, with `invalidSrcButton` and `ftpSrcButton` we want to ensure that clicking them doesn't actually change the binding of src.

What you can do here is click a button, wait for bind to apply the changes, use `expect` to see if the src is correct and then repeat for each one.